README.md

```diff
@@ -1,11 +1,17 @@
 # Tier <!-- omit in toc -->
 
+- [About](#about)
 - [Install](#install)
   - [Homebrew (macOS)](#homebrew-macos)
   - [Binary (macOS, linux, Windows)](#binary-macos-linux-windows)
   - [Go (most operating systems and architectures)](#go-most-operating-systems-and-architectures)
 - [Further Reading](#further-reading)
 
+## About
+tier is a tool that lets you define and manage your SaaS application's pricing model in one place (pricing.json). Tier will handle setting up and managing Stripe in a way that is much more friendly for SaaS and consumption based billing models. Tier's SDK can then be implemented for access checking, metering/reporting, and more.
+
+[More detail and documentation is available here](https://tier.run/docs).
```

Comment:
```suggestion
[More detail and documentation is available on the Tier website.](https://tier.run/docs)
```

Best to avoid words like "here" and "click" in links.